promotion/suggestions.go

```diff
@@ -216,6 +217,21 @@ func (service *Service) Suggest(ctx context.Context, credentials []CredentialBin
 		return err
 	}
 
+	// Delete this section once the issue is completed
+	// https://github.com/brave-intl/bat-go/issues/263
+	if suggestion.OrderID != nil {
+		// TODO Should we iterate through the funding types like below and create transactions for each type?
```

Comment:
i would say having a similar structure as what is below makes sense. either that or error out if there is more than 1 and we don't want there to be.

Reply:
Moved it over https://github.com/brave-intl/bat-go/pull/264/commits/afba0039ca66688d6f6675850e093e1636b5a2a4